### ISSUE TITLE
Fall damage wont be applied

### ISSUE THREAD

**Issue by X39** [X39/XMS2#44]

**Comment by flaver12** [comment]
Should it be like when you fall  from to high, that you break your legs?

**Comment by X39** [comment]
yup

**Comment by flaver12** [comment]
So is it for you okay when i try to fix this issue?

**Comment by X39** [comment]
for sure it is
quite simple actually
the problem is currently the only EH used for damage calculation is the HitPart EH
but to get fall damage (and other env. dmg) you have to use the normal HandleDamage EH
just did not yet fixed it as its not rly the most important thing on any agenda i have

**Comment by X39** [comment]
allow me the question,
whats your interest in XMS2?

**Comment by X39** [comment]
The output of "HandleDamage" when falldamage is applied:
```

16:41:30 "[BIS_fnc_objectVar_obj1,"head",0.0351354,<NULL-object>,""]"
16:41:30 "[BIS_fnc_objectVar_obj1,"",0.90946,<NULL-object>,""]"
16:41:30 "[BIS_fnc_objectVar_obj1,"",0.176937,BIS_fnc_objectVar_obj1,""]"
16:41:30 "[BIS_fnc_objectVar_obj1,"head",0.0351354,BIS_fnc_objectVar_obj1,""]"
16:41:30 "[BIS_fnc_objectVar_obj1,"body",0.757884,BIS_fnc_objectVar_obj1,""]"
16:41:30 "[BIS_fnc_objectVar_obj1,"hands",0.173226,BIS_fnc_objectVar_obj1,""]"
16:41:30 "[BIS_fnc_objectVar_obj1,"legs",0.45473,BIS_fnc_objectVar_obj1,""]"
```

**Comment by flaver12** [comment]
Sry for my late answer, is was busy at the weekend with some little webprojects^^.
So my interest in XMS2 is to help to bring the project back. 
I think that this medic system can be very good and very realistic.
I also want be made my on but the I realized that is too much work for me.
So i thought why you dont help a guy how have a great code base for a medical mod.
I hope i answered your question :) 

Back to the issue, I will look at that at tuesday if its okay for you?
Can we add, lets say for the car crash a whiplash?
I mean we have the info if the player is in a car and i think we can get the speed too.
https://community.bistudio.com/wiki/speed

**Comment by X39** [comment]
actually you dont have to get that info due to the fact that the car crash is already taken into account of the damage value

**Comment by flaver12** [comment]
I started with this issue today and i become this error msg:

![capture](https://cloud.githubusercontent.com/assets/5447323/9056742/6e93f7e0-3a95-11e5-9233-acd753015cb9.PNG)

I just build the addon with the addon builder.
Any ideas?

**Comment by X39** [comment]
check your Functions folder
i renamed the initMod.sqf file in a commit and it looks like your sync did not updated it correctly

**Comment by X39** [comment]
fixed in 7506b193054e966d908e67f1ae0e4797ebc4887b
